core/domain/user_id_migration.py

```diff
@@ -465,6 +469,12 @@ def _migrate_model(snapshot_model, commit_model_class):
                 'rights-%s-%s' % (
                     snapshot_model.get_unversioned_instance_id(),
                     snapshot_model.get_version_string()))
+            if commit_log_model is None:
+                snapshot_model.put(update_last_updated_time=False)
+                snapshot_class_name = snapshot_model.__class__.__name__
+                return (
+                    'SUCCESS_MISSING_COMMIT - %s' % snapshot_class_name,
```

Comment:
Is a missing commit actually a success? Should we be logging those cases?

Reply:
Success in the sense that we can fix this after the migration.